net/net.sh

```diff
@@ -76,6 +76,9 @@ Operate a configured testnet
                                       - If set, validators will skip verifying
                                         the ledger they already have saved to disk at
                                         boot (results in a much faster boot)
+   --persist-transaction-status
```

Comment:
🤔 linking this setting to the presence of `--rpc-port` is probably fine, and avoids us having to deal with support questions about how the RPC API isn't returning all the info

Reply:
This means that testnet and localnet nodes will have the persistent tx status store enabled by default. We might want more flexibility to limit rocksdb size at some point. But future Tyera can handle that.

Reply:
Well, enabling the persistent transaction store in local_cluster caused some instability related to this bug (https://github.com/rust-lang/rust/issues/39364). Thought my shortest path to victory was to put a disable arg in the Validator struct. Do you hate it?